internal/core/ports/room_ports.go

```diff
@@ -24,11 +24,23 @@ package ports
 
 import (
 	"context"
+	"sync"
 	"time"
 
+	"github.com/topfreegames/maestro/internal/core/entities"
 	"github.com/topfreegames/maestro/internal/core/entities/game_room"
 )
 
+type RoomManager interface {
+	DeleteRoomAndWaitForRoomTerminated(ctx context.Context, gameRoom *game_room.GameRoom) error
```

Comment:
Also, could we add comments describing these interface methods as we are doing with other primaries/secondary ports?